elasticgraph-query_registry/spec/acceptance/query_registry_spec.rb

```diff
@@ -104,6 +104,100 @@ module ElasticGraph
       }.to log_warning a_string_including("is unregistered")
     end
 
+    it "logs query_registered=true when a query is registered for the client" do
```

Comment:
```suggestion
    it "logs `query_registered: true` when a query is registered for the client" do
```

/nit I think it's slightly more readable this way--and ultimately it's logged as JSON which uses `key: value` syntax instead of `key=value`.

(And if you accept this suggestion, please apply it below...)